website/pages/_app.js

```diff
@@ -39,7 +39,7 @@ export default function App({ Component, pageProps }) {
         />
       </HashiHead>
       {ALERT_BANNER_ACTIVE && (
-        <AlertBanner {...alertBannerData} product="consul" />
+        <AlertBanner {...alertBannerData} product="consul" hideOnMobile />
```

Comment:
To match previous behaviour due to global styles previously shipped as part of `docs-page`, alert-banner is hidden on mobile.